content/SCALE/SCALEUIReference/Shares/iSCSISharesScreens.md

```diff
@@ -58,8 +58,8 @@ To display the **iSCSI Group** settings, click **Add**.
 {{< truetable >}}
 | Setting | Description |
 |---------|-------------|
-| **Portal Group ID** | Required. Select the number of the existing portal to use or leave it empty. |
-| **Initiator Group ID** | Select the existing initiator group ID that has access to the target from the dropdown list of options. **None**, **1(init1)**, or **3(ALL initiators Allowed)**. |
+| **Portal Group ID** | Required if specifying an iSCSI Group. Select the number of the existing portal to use. This is the portal group ID created in **Portals**. |
+| **Initiator Group ID** | Select the existing initiator group ID that has access to the target from the dropdown list of options. Options are **None**, **1(init1)**, or **3(ALL initiators Allowed)**. This is the initiator group ID created in **Initiator Groups**. |
```

Comment:
I would be wary about including this line:
> Options are **None**, **1(init1)**, or **3(ALL initiators Allowed)**

as these are only the options on a system where these particular Initial Groups have been defined.  They are **NOT** the options on any of my systems.

Reply:
Reworked the text here to be more generic.

Reply:
Added a new screenshot an adjusted this content.